packages/rpc/src/binance/Query.ts

```diff
@@ -0,0 +1,18 @@
+export class Query {
+    rpcUrl: string;
+
+    constructor(rpcUrl: string) {
+        this.rpcUrl = rpcUrl;
+    }
+
+    getNodeInfo = () => this.uri(`v1/node-info`);
+    getAccount = (address: string) => this.uri(`v1/account/${address}`);
+    getMarket = (limit?: number, offset?: number) => this.uri(`v1/markets?limit=${(limit == undefined)? 100 : limit}&offset=${(offset == undefined)? 0 : offset}`);
```

Comment:
Use default values instead of checking for undefined.

Reply:
Working on that now. Thank you